src/libs/actions/IOU.js

```diff
@@ -25,6 +26,14 @@ import ROUTES from '@src/ROUTES';
 import * as Policy from './Policy';
 import * as Report from './Report';
 
+let betas;
+Onyx.connect({
+    key: ONYXKEYS.BETAS,
+    callback: (val) => {
+        betas = val || [];
+    },
```

Comment:
NAB, is this for consistency?
```suggestion
    callback: (val) => betas = val || [],
```